src/yang/ietf-bgp-policy.yang

```diff
@@ -248,6 +248,32 @@ module ietf-bgp-policy {
           }
         }
       }
+      container next-hop-sets {
+        description
+          "Definition of a list of IPv4 or IPv6 next-hops which can
+           be matched in a routing policy.";
+
+        list next-hop-set {
+          key "name";
+          description
+            "List of defined next-hop sets for use in policies.";
+
+          leaf name {
+            type string {
+              length "1..79";
+            }
+            mandatory true;
+            description
+              "Name of the next-hop set.";
+          }
+          leaf-list address {
+            type bgp-next-hop-type;
+            max-elements 32;
```

Comment:
Why is this constrained?

Reply:
Again, this is what is in the OC model.

Reply:
We aren't OC.  IETF shouldn't be setting arbitrary limits on every BGP user on the planet.  Vendors are welcome to add their own constraints here.

I'm still angry at all of the MPLS MIBs that have Cisco limitations in them that mucked up implementations I've worked on.